tests/quick.rs

```diff
@@ -781,6 +801,11 @@ mod decompress_command {
 
         let mut cmd = command();
 
+        let input_metadata = std::fs::metadata(&sample1_tar_bz2).unwrap();
+
+        // sleep so that we'd notice if the timestamp was not set correctly
+        std::thread::sleep(Duration::from_secs(1));
```

Comment:
Maybe sleep 2 seconds to account for rounding errors?

Reply:
I'm assuming it just truncates?

Reply:
Getting the timestamp does indeed truncate. Hypothetically it would be possible for the OS to sleep slightly less than 1 second. If that happens and you got the initial timestamp right at the start of the second, it may be possible to get the second timestamp just before the next second. It probably won't happen in practice, but if it ever does it would be a heisenbug and likely hard to find the root cause, so I did somewhat prefer if we sleep a bit longer than 1s. Even if just 1.1s.